resources/assets/less/app/kanban.less

```diff
@@ -54,6 +54,110 @@
   }
 }
 
+
+/* navpill */
+.kanban-nav {
+	position: relative;
+	margin: 0px 0;
+}
+.kanban-pill-tag {
+	color: #afafaf;
+	padding-left: 10px !important;
+	padding-right: 10px !important;
+}
+.kanban-nav ul {
+	margin: 0;
+	padding: 0;
+}
+.kanban-nav li {
+	margin: 0 5px 10px 0;
+	padding: 0;
+	list-style: none;
+	display: inline-block;
+}
+.kanban-nav a {
+	padding: 3px 12px;
+	text-decoration: none;
+	color: #3B4252;
+	line-height: 100%;
+}
+.kanban-nav a:hover {
+	color: #000;
+}
+.kanban-nav .current a {
+	background: #3B4252;
+	color: #fff;
+	border-radius: 5px;
+}
+
+@media screen and (max-width: 600px) {
```

Comment:
I took this from a sample implementation I found and adapted it here.
I think there are some more possibilities like in SCSS here. Paremeters like @small-devices or something. And I know that the responsiveness of this menu is far from perfect yet.